src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/common/DashboardStepDef.scala

```diff
@@ -21,7 +21,7 @@ import uk.gov.hmrc.test.ui.pages.common.DashboardPage
 
 class DashboardStepDef extends BaseStepDef {
 
-  And("""^I should land on Dashboard page""")(() => DashboardPage.checkPage())
```

Comment:
You should not change `checkPage` to `checkDashboardPage`. This method override a method in `BasePage`.
 In addition, it is superfluous as due to the file name, `DashboardPage, we already now what page we are checking.